terraso_backend/apps/project_management/migrations/0017_auto_20230915_1527.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.5 on 2023-09-15 15:27
```

Comment:
question: could these migrations all be one migration by appending their operation lists? or is there a django reason they need to be separate migrations?

Reply:
I followed the [migration docs](https://docs.djangoproject.com/en/4.2/howto/writing-migrations/#migrations-that-add-unique-fields) here for adding the unique `OneToOneField`. I think you're right though, they probably could be all a single migration. I think the docs suggest doing it this way to have the data migration separate from the schema migration.